telethon/client/updates.py

```diff
@@ -311,6 +335,7 @@ def _process_update(self: 'TelegramClient', update, others, entities=None):
         if self._dispatching_updates_queue is None:
             task = self._loop.create_task(self._dispatch_update(*args))
             self._updates_queue.add(task)
+            task.add_done_callback(lambda _: None if task.cancelled() else task.exception())
```

Comment:
This is just silencing the exceptions, again, not sure that's the right way to do this. At the very least we should probably log them, much like they're logged when a handler throws an exception.

Reply:
@Lonami I added this callback because the first callback removes future from the set when future is done without retrieving possible exceptions. In my case there were futures which were created inside the library by some internal requests (like `get_me()` somewhere or whatever), that means that I have no public to those to retrieve exceptions from those.

Again here should be some mechanism either to silence exceptions or to let user handle it (or properly handle exceptions in internal futures of the lib).